### PR TITLE
Upgrade to 4.6.1

### PR DESCRIPTION
**Pull request by Mcastres**

### CODE REVIEW

File: api/package.json [hunk 1]
```diff
@@ -33,7 +31,7 @@
     "name": "A Strapi developer"
   },
   "strapi": {
-    "uuid": "FOODADVISOR-LOCAL-91bcb9c3-39e5-43e4-ac15-05df0813913f"
+    "uuid": "91bcb9c3-39e5-43e4-ac15-05df0813913f"
```
Comment (ivanThePleasant) [inline api/package.json:34]: Don't we still want some sort of project identifier here?

Comment (Mcastres) [reply api/package.json:34]: The `yarn seed` is the one that actually fill this:

```
 if (packageJSON.strapi.uuid.includes('FOODADVISOR')) return null;

      packageJSON.strapi.uuid =
        `FOODADVISOR-${
          process.env.GITPOD_WORKSPACE_URL ? 'GITPOD-' : 'LOCAL-'
        }` + uuid();

      const data = JSON.stringify(packageJSON, null, 2);
      fse.writeFileSync(filePath, data);
```

**Comment by ivanThePleasant** [comment]
LGTM 👍

**Review by ivanThePleasant — APPROVED** [review]

**Comment by Mcastres** [comment]
Thanks @ivanThePleasant 🙏

**Pull request merged**